crates/common/src/provider.rs

```diff
@@ -210,38 +214,13 @@ impl ProviderBuilder {
             jwt,
         ));
 
-        let is_local = is_local_endpoint(url.as_str());
-
         if is_local {
             provider = provider.interval(DEFAULT_LOCAL_POLL_INTERVAL);
         } else if let Some(blocktime) = chain.average_blocktime_hint() {
             provider = provider.interval(blocktime / 2);
         }
-        Ok(provider)
-    }
-}
 
-impl<'a> From<&'a str> for ProviderBuilder {
-    fn from(url: &'a str) -> Self {
-        Self::new(url)
-    }
-}
-
-impl<'a> From<&'a String> for ProviderBuilder {
-    fn from(url: &'a String) -> Self {
-        url.as_str().into()
-    }
-}
-
-impl From<String> for ProviderBuilder {
-    fn from(url: String) -> Self {
-        url.as_str().into()
-    }
-}
-
-impl<'a> From<Cow<'a, str>> for ProviderBuilder {
-    fn from(url: Cow<'a, str>) -> Self {
-        url.as_ref().into()
+        Ok(provider)
     }
 }
```

Comment:
I guess these were never used and since we're transitioning to alloy, makes sense to get rid of them